environment_setup/setup_software.sh

```diff
@@ -52,16 +50,14 @@ host_software_packages=(
     protobuf-compiler # This is required for the "NanoPb" library, which does not
                       # properly manage this as a bazel dependency, so we have
                       # to manually install it ourselves
-    python3       # Python 3
-    python3-dev # Python 3 headers
-    python3-venv # Virtual Environment
+    python3.8       # Python 3
+    python3.8-dev # Python 3 headers
+    python3.8-venv # Virtual Environment
```

Comment:
Were higher versions of Python not simple to integrate? (I noticed the previous comments mentioned 3.11)

Reply:
I tried python3.11 but updating dependencies was annoying so I stuck with 3.8